mne/utils/tests/test_progressbar.py

```diff
@@ -25,6 +26,18 @@ def iter_func(a):
             pass
     pytest.raises(Exception, iter_func, ProgressBar(20))
 
+    # Make sure different progress bars can be used
+    with catch_logging() as log, modified_env(MNE_TQDM='tqdm'), \
+            use_log_level('debug'), ProgressBar(np.arange(3)) as pbar:
+        for p in pbar:
+            pass
```

Comment:
I'm not familiar with this part of MNE or TQDM at all; what is this look intended to achieve? :)

Reply:
When you iterate over the progressbar it emits its messages, so this is just to get it to do that / actually use the object

Reply:
Thanks for the explanation!